docs/develop/python/message-passing.mdx

```diff
@@ -130,76 +142,95 @@ class GreetingWorkflow:
             raise ValueError(f"{language.name} is not supported")
 ```
 
-- The decorator can take arguments. See the API reference docs: [`workflow.update`](https://python.temporal.io/temporalio.workflow.html#update).
+- The Update decorator can take arguments (like, `name`, `dynamic` and `unfinished_policy`) as described in the API reference docs for  [`workflow.update`](https://python.temporal.io/temporalio.workflow.html#update).
+
+- About validators:
+  - Use validators to reject an Update before it is written to History.
+    Validators are always optional.
+    If you don't need to reject updates, you can skip them.
+  - The SDK automatically provides a validator decorator named `@<update-handler-name>.validator`.
+    The validator must accept the same argument types as the handler and return `None`.
+    You can create your own validator with the same constraints by marking a method with the `@update_workflow_status.validator` decorator.
+  - For more flexibility, use code-based checks at the start of the Update handler instead of validators.
+    Raise an `ApplicationError` for any issues that would prevent the Update from running.
+    This approach adds rejected Update requests to your Event History if exceptions occur.
```

Comment:
I think there was some confusion in our discussion in another channel; let's delete all this:

```suggestion
```

Reply:
Removed